### PR TITLE
sponsor-text template, cta, icon image, wordpress support

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: app/views/ad_templates/sponsored-text/themes/dark.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#cf {
+  text-decoration: none;
+  color: #aaaaaa;
+  line-height: 1.7;
+  font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
```
Comment (hopsoft) [inline app/views/ad_templates/sponsored-text/themes/dark.css]: Does unquoted `Helvetica Neue` work correctly? I thought we deleted that entry from the other stylesheets.

Comment (coderberry) [reply app/views/ad_templates/sponsored-text/themes/dark.css]: According to the browser, it works

File: app/controllers/wordpress_snippets_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          csv << column_names
+          @countries.each do |x|
+            csv << x.values
+          end
+        end
+        send_data results, filename: "pricing-#{Date.today}.csv"
+      end
+    end
+  end
+end
```
[the remark this answers is not in the document]
Comment (coderberry) [reply app/controllers/wordpress_snippets_controller.rb]: I don't believe so. It's public data that we will show on wordpress

File: app/controllers/creatives_controller.rb [hunk 3]
```diff
@@ -72,10 +72,10 @@ def set_creative
   end
 
   def creative_params
-    params.require(:creative).permit(:name, :headline, :body)
+    params.require(:creative).permit(:name, :headline, :body, :cta)
   end
 
   def creative_image_params
-    params.require(:creative).permit(:small_blob_id, :large_blob_id, :wide_blob_id)
+    params.require(:creative).permit(:icon_blog_id, :small_blob_id, :large_blob_id, :wide_blob_id)
```
[the remark this answers is not in the document]
Comment (coderberry) [reply app/controllers/creatives_controller.rb]: I'll fix this

**Pull request merged**